deploy/README.md

```diff
@@ -2,19 +2,19 @@
 
 A guide for deploying web apps in a sane and repeatable way
 
-**NOTE:** everything described here should be replicated in the [example app](https://github.com/groupbuddies/gb-puppet-test-app)
+**NOTE:** everything described here should be replicated in the [example app](https://github.com/subvisual/gb-puppet-test-app)
 
 # Server setup
 
 * Use [DigitalOcean](https://www.digitalocean.com/). Be sure to select the `Enable Backups` feature, which can only be enabled during creation
 * Use Ubuntu 12.04 (Precise Pangolin) x64, unless you have specific reasons to do otherwise (upstart has issues in later versions)
-* Provision the server using [GB Puppet](https://github.com/groupbuddies/gb-puppet). Follow the README there for the appropriate steps.
+* Provision the server using [Subvisual Puppet](https://github.com/subvisual/gb-puppet). Follow the README there for the appropriate steps.
```

Comment:
We need to start somewhere. I'm actually going to change the name of the repo after finishing some other tasks.